src/Microsoft.AspNetCore.SignalR.Core/Hub.cs

```diff
@@ -55,20 +67,34 @@ public IGroupManager Groups
             }
         }
 
+        /// <summary>
+        /// Called when the hub is connected to.
+        /// </summary>
+        /// <returns>A <see cref="Task"/> that represents the asynchronous connect.</returns>
         public virtual Task OnConnectedAsync()
         {
             return Task.CompletedTask;
         }
 
+        /// <summary>
+        /// Called when the hub is disconnected from.
```

Comment:
Called when **a connection with** the hub is **terminated** ~~disconnected from~~.